sweet.up/DiscoverAndMatch/categories.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  categories.swift
+//  sweet.up
+//
+//  Created by Tomasz Ferens on 25/10/2020.
+//
+
+import Foundation
+
+struct Category: Identifiable {
+  var id = UUID()
+
+  var name: String
+  var textOnImage: String
+  var description: String? = nil
+}
+
+let categories: [String: Category] = [
```

Comment:
So this constant is automatically accessed in whole project ?  :D

Reply:
yes :D